support/cas-server-support-consent-webflow/src/main/java/org/apereo/cas/web/consent/config/CasConsentReviewConfiguration.java

```diff
@@ -0,0 +1,138 @@
+package org.apereo.cas.web.consent.config;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apereo.cas.authentication.principal.Service;
+import org.apereo.cas.authentication.principal.ServiceFactory;
+import org.apereo.cas.authentication.principal.WebApplicationService;
+import org.apereo.cas.web.pac4j.CasSecurityInterceptor;
+import org.apereo.cas.configuration.CasConfigurationProperties;
+import org.apereo.cas.services.RegexRegisteredService;
+import org.apereo.cas.services.ServicesManager;
+import org.apereo.cas.web.consent.CasConsentReviewController;
+import org.apereo.cas.CasProtocolConstants;
+import org.apereo.cas.consent.ConsentEngine;
+import org.apereo.cas.consent.ConsentRepository;
+import org.pac4j.core.authorization.authorizer.IsAuthenticatedAuthorizer;
+import org.pac4j.core.config.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
+import org.springframework.cloud.context.config.annotation.RefreshScope;
+import org.springframework.web.servlet.HandlerInterceptor;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.view.RedirectView;
+
+import javax.annotation.PostConstruct;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This is {@link CasConsentReviewConfiguration}.
+ *
+ * @author Arnold Bergner
+ * @since 5.2.0
+ */
+@Configuration("casConsentReviewConfiguration")
+@ConditionalOnBean(name = "casSecurityContextConfiguration")
+public class CasConsentReviewConfiguration extends WebMvcConfigurerAdapter {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CasConsentReviewConfiguration.class);
+
+    private static final String CAS_CLIENT_NAME = "CasClient";
+
+    @Autowired
+    private CasConfigurationProperties casProperties;
+
+    @Autowired
+    @Qualifier("config")
```

Comment:
This probably needs to be renamed. `config` is way too generic and easily at conflict with something else.